lnwallet/interface.go

```diff
@@ -35,6 +35,13 @@ const (
 	PubKeyHash
 )
 
+var (
```

Comment:
Looks like something was left off this commit? I only see a single error type being added.

Reply:
The `godoc` on the interface method should be updated to detail the expected return values in the case of an error (basically these types).

Reply:
I originally had more error types defined, but it was hard to unify them across the three backends.